src/workingtitle-vcockpits-instruments-airliners/html_ui/Pages/VCockpit/Instruments/Airliners/Shared/WT/AltimeterIndicator.js

```diff
@@ -107,8 +107,8 @@ class Jet_PFD_AltimeterIndicator extends HTMLElement {
         else {
             Utils.RemoveAllChildren(this.rootGroup);
         }
-        if (!this.pressureSVG)
-            this.pressureSVG = document.createElementNS(Avionics.SVG.NS, "text");
+        if (!this.pressureSVGLeftPart)
+            this.pressureSVGLeftPart = document.createElementNS(Avionics.SVG.NS, "text");

```

Comment:
@flymypi I was not sure what I should test for here; chose just to test for LeftPart. Do I need to test for all three parts (left, center, right)?

Reply:
Additionally, I think that there is a bug in this test in the current release. See issue #1160 I opened a while back.

Reply:
Your fix for issue #1160 makes that entire check a bit redundant because you create that text-element anyway in line 181: `this.pressureSVGLeftPart = document.createElementNS(Avionics.SVG.NS, "text");`

Because it is now seperated into three parts and the construct should only run once, it's cleaner to just set create those elements before attaching the attributes. So I suggest you just remove this check.